pyccl/pk2d.py

```diff
@@ -365,9 +393,6 @@ def __add__(self, other):
 
         return new
 
-    def __radd__(self, other):
-        return self.__add__(other)
-
     def __mul__(self, other):
         """Multiply two Pk2D instances.
```

Comment:
Can we document that the second one can actually be a number?

Reply:
done in the class docstring